data/sql_updates/large_aggregates.sql

```diff
@@ -142,11 +146,14 @@ party_totals as (
 -- Independent expenditure only
 ie as (
     select
-        extract(month from to_date(cast(coverage_end_date as text), 'YYYY-MM-DD')) as month,
-        extract(year from to_date(cast(coverage_end_date as text), 'YYYY-MM-DD')) as year,
-        sum(independent_expenditures_period) as receipts,
-        sum(independent_contributions_period) as disbursements
-    from ofec_reports_ie_only_mv_tmp
+        extract(month from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD')) as month,
+        extract(year from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD')) as year,
+        sum(ttl_indt_exp) as receipts,
+        sum(ttl_indt_contb) as disbursements
+    from
+        fec_vsum_f5
+    where
```

Comment:
Good catch. I think I changed the source of the table and then did then looked for duplicates and that made it better, but there was not a need to change tables. I can change it back, but I need to keep working on it anyway.